external_projects/bccm_player/android/src/main/kotlin/media/bcc/bccm_player/ExoPlayerController.kt

```diff
@@ -65,11 +65,12 @@ class ExoPlayerController(private val context: Context) : PlayerController() {
         player = BccmForwardingPlayer(this)
 
         val youboraOptions = Options()
-        setBasicYouboraOptions(youboraOptions, BccmPlayerPluginSingleton.npawConfigState.value)
         youboraOptions.isAutoDetectBackground = false
         youboraOptions.userObfuscateIp = true
         youboraOptions.isEnabled = false
         youboraOptions.isParseManifest = true
+        setBasicYouboraOptions(youboraOptions, BccmPlayerPluginSingleton.npawConfigState.value)
```

Comment:
this is actually not relevant to the issue, but I just moved it because it should be here